### PR TITLE
Add raml-cop to Travis build

### PR DESCRIPTION
**Pull request by cherewaty**
screen shot 2018-06-27 at 3 16 23 pm

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,9 @@ rvm:
 
 install:
   - bundle install
+  - yarn global add raml-cop
```
Comment (sduvvuri-ebsco) [inline .travis.yml:11]: This might be a really dumb question but can we add raml-cop using yarn? I have seen it to be part of npm?

**Review by sduvvuri-ebsco — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (sduvvuri-ebsco) [reply .travis.yml:11]: never mind, just saw your screenshot

Comment (cherewaty) [reply .travis.yml:11]: Any `npm` package can be installed with `yarn`. `yarn` is really just a drop in replacement for the `npm` CLI - they talk to the same registry through different proxies.

Comment (sduvvuri-ebsco) [reply .travis.yml:11]: Ah, good to know. Did not know that, thanks @cherewaty !

**Pull request merged**